### ISSUE TITLE
Crash: RadialGradient using radius of 0

### ISSUE THREAD

**Issue by Czino** [SrBrahma/react-native-shadow-2#47]
I found that the `radialGradient` function crashes when passed with a `shadowRadius` of 0

```
function radialGradient(id, top, left, radius, shadowRadius) {
  return (<RadialGradient id={id} cx={left ? shadowRadius : 0} cy={top ? shadowRadius : 0} r={shadowRadius} ...
}
```

Error
```
 java.lang.IllegalArgumentException: radius must be > 0
        at android.graphics.RadialGradient.<init>(RadialGradient.java:104)
        at android.graphics.RadialGradient.<init>(RadialGradient.java:67)
        at com.horcrux.svg.Brush.setupPaint(Brush.java:213)
```

This is the component that makes it crash

```
<Shadow distance={0} radius={0} startColor="#E43B5F" finalColor="#E43B5F" />
```

I cleaned up my code to not having to encounter this case but I am still raising this issue because I think the package could handle it more gracefully, even if it's a warning or error log and fallback to sensible values.

**Comment by SrBrahma** [comment]
Many thanks for the report. I had some crashes when distance is 0, maybe this is what caused the crashes I had (the shadowRadius = radius + distance). I will investigate this asap.

I had planned to investigate this and fix it on 7.0.0, but I will release a hotfix for it as a patch to v6.

**Comment by SrBrahma** [comment]
Hi! I had the time to test the patch today.

Just released it in 6.0.5 via https://github.com/SrBrahma/react-native-shadow-2/commit/ce32eea4a17580447a0e75444881aed23c9b9e8a.

I would appreciate if you also test it and give me your feedback about it!

Thanks!

**Comment by Czino** [comment]
Just tested it, setting distance and radius explicitly to 0 doesn't result in a crash anymore
